src/PolicyInsights/PolicyInsights/Cmdlets/GetAzureRmPolicyState.cs

```diff
@@ -150,6 +149,10 @@ public class GetAzureRmPolicyState : PolicyInsightsCmdletBase
         [ValidateNotNullOrEmpty]
         public string Apply { get; set; }
 
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceScope, Mandatory = false, HelpMessage = ParameterHelpMessages.Expand)]
+        [ValidateNotNullOrEmpty]
+        public string Expand { get; set; }
```

Comment:
Please update help using this doc: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md

Please also link to your design review here: https://github.com/Azure/azure-powershell-cmdlet-review-pr, which is required for any API surface change.

Reply:
@maddieclayton Both are done.